content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+  requirements over time
+---
+
+> **Early Access**
+>
+> Docker Scout is in [early access](/release-lifecycle/#early-access-ea). The
+> policy feature is a closed access feature of Docker Scout, and is only
+> available to organizations participating in the early access program for
+> policies.
+>
+> If you're interested in trying out this feature, reach out using the form on
+> the [Docker Scout product page](https://docker.com/products/docker-scout)
+{ .restricted }
+
+In software supply chain management, maintaining the security and reliability
+of artifacts is a top priority. Policy in Docker Scout introduces a layer of
+control, on top of existing analysis capabilities. It lets you define supply
+chain rules for your artifacts, and helps you track how your artifacts perform,
+relative to your rules and thresholds, over time.
+
+Learn how you can use policy to ensure that your artifacts align with your
+desired standards.
+
+## Policy evaluation
+
+When you activate Docker Scout for a repository, image tags that you push are
+[automatically analyzed](../advanced-image-analysis.md). The analysis gives you
+insights about the composition of your images, including what packages that
+they contain and what vulnerabilities that they're exposed to. Policy builds on
+top of the image analysis feature, interpreting the analysis results against
+the rules defined by policies.
+
+A policy defines one or more criteria that your artifacts should fulfill. For
+example, one of the default policies in Docker Scout is the **Critical
+vulnerabilities** policy, which proclaims that your artifacts must not contain
+any critical vulnerabilities. If an artifact contains one or more
+vulnerabilities with a critical severity, that artifact will have failed the
+policy check.
+
+In Docker Scout, policies are not meant to be used for gate-keeping. The
+purpose of policies is to help you ratchet forward your security and supply
+chain stature. For artifacts that fail the policy check, you can track how the
+fail gap changes over time. This lets you see if your artifact is improving or
+deteriorating relative to policy.
+
+Policies don't necessarily have to be related to application security and
+vulnerabilities. You can use policies to measure and track other aspects of
+supply chain management as well, such as base image dependencies and
+open-source licenses.
+
+## Default policies
+
+Docker Scout ships the following three out of the box policies:
+
+- [Critical and high vulnerabilities with fixes](#critical-and-high-vulnerabilities-with-fixes)
+- [Critical vulnerabilities](#critical-vulnerabilities)
+- [Packages with GPL3+ licenses](#packages-with-gpl3-licenses)
+
+These policies are turned on for your active repositories by default. There's
+currently no way to turn off or configure these policies.
+
+### Critical and high vulnerabilities with fixes
+
+This policy requires that your artifacts aren't exposed to known
```

Comment:
This policy also has the concept of age. It only considers vulnerabilities that are >30 days old. The idea being that a brand new vulnerability shouldn't necessarily break policy until a reasonable time has passed for the teams in question to address it.

Reply:
>30 days since what? Since it was first disclosed in one of the sources that we track? Or since it was last updated? Or since it was first detected in an analysis?

Reply:
I believe it's "Since it was first disclosed in one of the sources that we track" but will let Chris confirm

Reply:
Yep, that's correct. Since it was first published.